### PR TITLE
fix: bugs in organization screen and repository issues list

### PR DESCRIPTION
**Pull request by chinesedfan**
| Question         | Response    |
| ---------------- | ----------- |
| Version?         | v1.6.0      |
| Devices tested?  | iPhone 6 Simulator |
| Bug fix?         | yes      |
| New feature?     | no      |
| Includes tests?  | no      |
| All Tests pass?  | yes      |
| Related ticket?  | #826        |

---

## Screenshots

| Before   | After    |
| -------- | -------- |
|image|image|
|image|image|

## Description

Fixed 2 bugs,

- https://github.com/gitpoint/git-point/issues/826#issuecomment-427588684
- in any repository, i.e. gitpoint/git-point, click `view all` of preview issues, click any issue of the opened issue list

### CODE REVIEW

File: src/organization/screens/organization-profile.screen.js [hunk 1]
```diff
@@ -143,6 +143,7 @@ class OrganizationProfile extends Component {
             <MembersList
               title={t('MEMBERS', locale)}
               members={orgMembers}
+              noMembersMessage={t('No contributors found', locale)}
```
Comment (andrewda) [inline src/organization/screens/organization-profile.screen.js]: For an org, this seems a little better fit:

```suggestion
              noMembersMessage={t('No members found', locale)}
```

File: src/locale/languages/zhCn.js [hunk 2]
```diff
@@ -89,6 +89,7 @@ module.exports = {
   'No contributors found': '没有贡献者',
   'No description provided.': '没有提供简介',
   'No issues': '没有问题',
+  'No members found': '',
```
Comment (lex111) [inline src/locale/languages/zhCn.js]: @chinesedfan can you provide a translation here?

**Comment by lex111** [comment]
Added translation of the new phrase into Russian and Ukrainian.

File: src/locale/languages/zhCn.js (hunk 2 above)
Comment (chinesedfan) [reply src/locale/languages/zhCn.js]: @lex111 Done. And thanks for your translations.

File: src/locale/languages/es.js [hunk 3]
```diff
@@ -90,6 +90,7 @@ module.exports = {
   'No contributors found': 'No se encontraron contribuidores',
   'No description provided.': 'No se ha proporcionado ninguna descripción.',
   'No issues': 'No hay issues',
+  'No members found': '',
```
Comment (lex111) [inline src/locale/languages/es.js]: friendly ping @alejandronanez 
Can you translate a new phrase?

Comment (alejandronanez) [reply src/locale/languages/es.js]: :wave: @lex111,

I think this could work.

```suggestion
'No members found': 'No hay miembros'
```

Comment (lex111) [reply src/locale/languages/es.js]: That's right, this is a new feature of GitHub, thanks!

**Comment by chinesedfan** [comment]
@lex111 Fixed. It is due to the new Catalan translation doesn't contain a key.

File: src/locale/languages/ca.js [hunk 4]
```diff
@@ -91,6 +91,7 @@ module.exports = {
   'No contributors found': "No s'ha trobat cap contribuïdor",
   'No description provided.': "No s'ha trobat cap descripció",
   'No issues': 'No hi ha ninguna issue',
+  'No members found': '',
```
[the remark this answers is not in the document]
Comment (olegkorol) [reply src/locale/languages/ca.js]: Done @lex111. 
Also added a German translation (I saw it was missing as well) :)

[the remark this answers is not in the document]
Comment (lex111) [reply src/locale/languages/ca.js]: Awesome, thanks! :heart_eyes:

File: src/locale/languages/fr.js [hunk 5]
```diff
@@ -92,6 +92,7 @@ module.exports = {
   'No contributors found': 'Aucun contributeur trouvé',
   'No description provided.': 'Aucune description fournie.',
   'No issues': 'Aucun ticket',
+  'No members found': '',
```
Comment (lex111) [inline src/locale/languages/fr.js]: Bonjour @machour! :smile: 

Can you please add a translation of this new phrase?

File: src/locale/languages/nl.js [hunk 6]
```diff
@@ -90,6 +90,7 @@ module.exports = {
   'No contributors found': 'Geen medewerkers',
   'No description provided.': 'Niet voorzien van een beschrijving.',
   'No issues': 'Geen issues',
+  'No members found': '',
```
Comment (lex111) [inline src/locale/languages/nl.js]: Hi @RolfKoenders 

Can you please add a translation of this new phrase?

File: src/locale/languages/fr.js (hunk 5 above)
Comment (olegkorol) [reply src/locale/languages/fr.js]: `Aucun membre trouvé`? :D @lex111

Comment (machour) [reply src/locale/languages/fr.js]: Perfect @olegkorol :)

File: src/locale/languages/nl.js (hunk 6 above)
Comment (chinesedfan) [reply src/locale/languages/nl.js]: @RolfKoenders I will merge it first. And anybody can feel free to translate those missing words later.

**Pull request merged**